lbz/authz.py

```diff
@@ -2,10 +2,12 @@
 """
 Authorization module.
 """
+# TODO: standardize the naming convention
+# In some places, we are using permission and in other resource but it looks like sometimes they mean the same
```

Comment:
Keep the limit of signs and also, it would be good to put this TODO closer to the place where the problem is visible:
```python
def _check_resource(self, resource: Union[dict, str]) -> None:
    # TODO: standardize the naming convention (resource != permission)
    self._deny_if_all(resource)
```